src/services/notifications/notification_service.cpp

```diff
@@ -6,6 +6,16 @@ namespace big
 {
 	notification_service::notification_service()
 	{
+		// TODO (maybe): Exception to be checked, "std::vformat" formatting exception
+		// Dumping stacktrace:
+		// [0]     KERNELBASE.dll RaiseException
+		// [1]     D:\a\_work\1\s\src\vctools\crt\vcruntime\src\eh\throw.cpp L: 75 _CxxThrowException
+		// [2]     C:\Program Files\Microsoft Visual Studio\2022\Enterprise\VC\Tools\MSVC\14.35.32215\include\format L: 79 std::_Throw_format_error
+		// [3]     C:\Program Files\Microsoft Visual Studio\2022\Enterprise\VC\Tools\MSVC\14.35.32215\include\format L: 942 std::_Fmt_codec<char,0>::_Find_encoded
+		// [4]     C:\Program Files\Microsoft Visual Studio\2022\Enterprise\VC\Tools\MSVC\14.35.32215\include\format L: 1293 std::_Parse_format_string<char,std::_Format_handler<char> &>
+		// [5]     C:\Program Files\Microsoft Visual Studio\2022\Enterprise\VC\Tools\MSVC\14.35.32215\include\format L: 3422 std::vformat_to<std::back_insert_iterator<std::basic_string<char,std::char_traits<char>,std::allocator<char> > > >
+		// [6]     C:\Program Files\Microsoft Visual Studio\2022\Enterprise\VC\Tools\MSVC\14.35.32215\include\format L: 3493 std::vformat<0>
+		// [7]     C:\Users\fehu\source\repos\YimMenu\src\services\notifications\notification_service.cpp L: 9 big::notification_service::notification_service
```

Comment:
why are you leaving a comment with an output error, also be aware this has your pc user name in it...